dcicutils/glacier_utils.py

```diff
@@ -388,21 +381,16 @@ def copy_object_back_to_original_location(self, bucket: str, key: str, storage_c
             else:
                 # Force copy the object into standard in a single operation
                 copy_source = {'Bucket': bucket, 'Key': key}
-                copy_args = {
+                copy_target = {
                     'Bucket': bucket, 'Key': key,
                     'StorageClass': storage_class,
                 }
                 if version_id:
                     copy_source['VersionId'] = version_id
-                    copy_args['CopySourceVersionId'] = version_id
+                    copy_target['CopySourceVersionId'] = version_id
                 if tags:
-                    copy_args['Tagging'] = tags
-                if self.kms_key_id:
-                    copy_args['ServerSideEncryption'] = 'aws:kms'
-                    copy_args['SSEKMSKeyId'] = self.kms_key_id
-                response = self.s3.copy_object(
-                    **copy_args, CopySource=copy_source
-                )
```

Comment:
These deletions in glacier utils should be reverted I think, as KMS Key args are definitely needed. Not sure why these deletions are here.

Reply:
Hm, not sure what this would have been about - I didn't make any changes related to glacier_utils.